test/ffmpeg-qsv/encode/vp9.py

```diff
@@ -125,3 +125,38 @@ def test(self, case, gop, bitrate, fps, slices, refmode, quality, looplvl, loops
     self.init(spec, case, gop, bitrate, fps, slices, quality)
     self.encode()
 
+class seek(VP9_8EncoderTest):
+  def init(self, tspec, case, rcmode, gop, bitrate, maxrate, fps, seek):
```

Comment:
Non-LP is skipped on most platforms... however, on MTL this will break:
```
Error: TypeError: seek.init() missing 1 required positional argument: 'seek'
```

Reply:
It has been fixed, thanks for the review.